test-cases/longevity/longevity-large-partition-4days.yaml

```diff
@@ -2,8 +2,14 @@ test_duration: 6480
 
 bench_run: true
 
-prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -max-rate=300 -replication-factor=3 -partition-count=10 -clustering-row-count=10000000 -clustering-row-size=5120 -concurrency=7 -rows-per-request=10"]
-stress_cmd: ["scylla-bench -workload=uniform -mode=read -replication-factor=3 -partition-count=10 -clustering-row-count=10000000 -clustering-row-size=5120 -rows-per-request=10 -concurrency=7 -max-rate=32000 -duration=5760m"]
+prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=20 -clustering-row-count=10000000 -clustering-row-size=2048 -concurrency=50000 -rows-per-request=2000 -timeout=180s -connection-count 1000",
```

Comment:
```suggestion
prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=20 -clustering-row-count=100000000 -clustering-row-size=500 -concurrency=500 -rows-per-request=500 -timeout=180s -connection-count 100",
```

Reply:
(In all commands of course)

Reply:
Take a live cluster, drop the table and start do experiments manually like the above, till you find an optimized workload where the prepare is faster and utilizes the cluster better.
If/when you reach to a point that it's not going more, you may won't to split the commands into 3 more commands during write.

Reply:
done